src/gluonts/time_feature/lag.py

```diff
@@ -122,6 +128,15 @@ def _make_lags_for_month(multiple, num_cycles=3):
             + _make_lags_for_day(offset.n / (60.0 * 24))
             + _make_lags_for_week(offset.n / (60.0 * 24 * 7))
         )
+    # second
+    elif offset_name == "S":
+        lags = (
+            _make_lags_for_second(offset.n)
+            + _make_lags_for_minute(offset.n / 60.0)
```

Comment:
Why are we using floats here at all? 🤔

Reply:
But `/` will already return a float, so the `0` is unneeded. I assume it's because this once was written for Py2.